PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,16 +33,16 @@ components, etc.
 please ask if you have any concerns about the risk of the PR.
 
 *Testing*:
->Explanation of what testing was done, for example: unit test, 
+>Explanation of what testing was done, for example: unit test,
 integration, manual testing, etc.
 
->Note: It isn’t expected to do all 
-forms of testing, please use your best judgement or ask for guidance 
+>Note: It isn’t expected to do all
+forms of testing, please use your best judgement or ask for guidance
 if you are unsure. A good rule of thumb is the riskier the change, the
 more comprehensive the testing should be.
 
 [optional *Deprecated*:]
 > Description of what is deprecated.
 
-[optional *Release Notes*:]
- > Description of what to add to next point release notes.
+[optional *Release Notes*:] If this change is user impacting you **must** add a release note to
+[RAW_RELEASE_NOTES.md](RAW_RELEASE_NOTES.md). Thank you!
```

Comment:
Would it be a bunch of work to have a presubmit which required you either edit RAW_RELEASE_NOTES or have metadata in your PR explicitly noting that you don't need to?  Might be worth a filed issue if we find ourselves forgetting.  presubmits like this are annoying as heck but do tend to catch a lot.

Reply:
There already is an optional metadata field in the commit for this, it's a question of enforcing.

Reply:
That sounds awesome, but I would rather walk before we run. I will open a follow up issue.

Reply:
Opened https://github.com/envoyproxy/envoy/issues/2133